packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.tsx

```diff
@@ -32,25 +35,30 @@ export const makeVVirtualScrollProps = propsFactory({
     type: Array as PropType<readonly unknown[]>,
     default: () => ([]),
   },
+  renderless: Boolean,
 
   ...makeVirtualProps(),
   ...makeComponentProps(),
   ...makeDimensionProps(),
 }, 'VVirtualScroll')
 
-export const VVirtualScroll = genericComponent<new <T>(
+export const VVirtualScroll = genericComponent<new <T, Renderless extends boolean = false>(
   props: {
     items?: readonly T[]
+    renderless?: Renderless
   },
   slots: {
-    default: VVirtualScrollSlot<T>
+    default: VVirtualScrollSlot<T> & (Renderless extends true ? {
+      itemRef: Ref<HTMLElement | undefined>
+    } : {})
   }
 ) => GenericProps<typeof props, typeof slots>>()({
   name: 'VVirtualScroll',
 
   props: makeVVirtualScrollProps(),
 
   setup (props, { slots }) {
+    const vm = getCurrentInstance('VVirtualScroll')
     const { dimensionStyles } = useDimension(props)
     const {
       containerRef,
```

Comment:
Not sure what you're suggesting, that's what it already does.

Reply:
If `computedItems` is tracked in runtime console log. You can see before `containerRef` is ready, `computedItems` was initially calculated based on window height. My screen is small, it renders 50 items. Big screens will hydrate more.

Maybe this is something "nice to not have" rather than a blocker.

![computed-items](https://github.com/vuetifyjs/vuetify/assets/5698884/35f756dc-fc1a-4972-b523-1a7837861943)

Reply:
Not a blocker, it doesn't have an element to get the height of initially.